draft-ietf-mls-protocol.md

```diff
@@ -1148,39 +1166,42 @@ struct {
         case 1: T value;
     }
 } optional<T>;
-
-struct {
-    HPKEPublicKey public_key;
-    uint32 unmerged_leaves<0..2^32-1>;
-    opaque parent_hash<0..255>;
-} ParentNode;
```

Comment:
This struct got deleted but is still referenced in several places

Reply:
Yeah, I think we still need this one.  It goes in the syntax for how you send the tree to a new member.